course/course.pot

```diff
@@ -0,0 +1,261 @@
+# Copyright (C) 2022 Automattic
```

Comment:
I think we don't check in pot files in this repo, since they are updated automatically on dotcom and would involve a reverse-syncing process that our systems are not well suited for.

Reply:
I'm a bit confused about this as it seems to conflict with some other advice I received on [Slack](https://a8c.slack.com/archives/C029FM1EH/p1668438008975119). How will it work on WP.org if we don't commit the .pot file?